src/dialogs/list-diagrams/list-diagrams.tsx

```diff
@@ -54,14 +69,19 @@ export const ListDiagrams: React.FC<Props> = ({
 
       <S.Content dividers>
         <S.ListDiagram>
-          {data.length > 0 &&
-            data?.map((diagram: any) => (
+          {orderData.length > 0 &&
+            orderData.map((diagram: any) => (
               <S.ItemDiagram disablePadding>

```

Comment:
Adicionar propriedade "key""

```suggestion
              <S.ItemDiagram key={diagram.id} disablePadding>

```